src/app/containers/StoryPromo/index.jsx

```diff
@@ -1,27 +1,38 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useContext } from 'react';
 import { shape } from 'prop-types';
-import { latin } from '@bbc/gel-foundations/scripts';
 import StoryPromoComponent, {
   Headline,
   Summary,
+  Link,
 } from '@bbc/psammead-story-promo';
 import { storyItem } from '../../models/propTypes/storyItem';
-import deepGet from '../../helpers/json/deepGet';
 import StoryPromoFigure from './Figure';
 import Timestamp from '../Timestamp';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import deepGet from '../../helpers/json/deepGet';
 
 const StoryPromo = ({ item }) => {
+  const { script } = useContext(ServiceContext);
```

Comment:
Just some thoughts. I think it is better to use context after the `headline` and `url` checks. That makes the definition of `script` closer to where it is defined. Makes it easy to read.

Reply:
Do you mean after the `if`?
```
if (!headline || !url) {
    return null;
  }
```
In that case, I would also put the check just after the `headline` and `url` declaration so if one of them is null we don't declare the rest.

Reply:
Yeah. After the `if`.

Reply:
This is what happens when you put it in that order:
<img width="703" alt="Screenshot 2019-05-24 at 10 45 33" src="https://user-images.githubusercontent.com/46446236/58319141-30d01d80-7e11-11e9-8e3f-0c7de350b027.png">